eurostag-converter/src/main/java/com/powsybl/eurostag/converter/EurostagEchExport.java

```diff
@@ -0,0 +1,938 @@
+    }
+
+    private double getR(TwoWindingsTransformer twt) {
+        return getValue(twt.getR(),
+                twt.getRatioTapChanger() != null ? twt.getRatioTapChanger().getCurrentStep().getR() : 0,
+                twt.getPhaseTapChanger() != null ? twt.getPhaseTapChanger().getCurrentStep().getR() : 0);
+    }
+
+    private double getG1(TwoWindingsTransformer twt) {
+        return getValue(config.isSpecificCompatibility() ? twt.getG() / 2 : twt.getG(),
+                twt.getRatioTapChanger() != null ? twt.getRatioTapChanger().getCurrentStep().getG() : 0,
+                twt.getPhaseTapChanger() != null ? twt.getPhaseTapChanger().getCurrentStep().getG() : 0);
+    }
+
+    private double getB1(TwoWindingsTransformer twt) {
+        return getValue(config.isSpecificCompatibility() ? twt.getB() / 2 : twt.getB(),
+                twt.getRatioTapChanger() != null ? twt.getRatioTapChanger().getCurrentStep().getB() : 0,
+                twt.getPhaseTapChanger() != null ? twt.getPhaseTapChanger().getCurrentStep().getB() : 0);
+    }
+
+    private void createTransformers(EsgNetwork esgNetwork, EsgGeneralParameters parameters) {
+        Set<String> additionalBanksIds = new HashSet<>();
+
+        for (TwoWindingsTransformer twt : Identifiables.sort(network.getTwoWindingsTransformers())) {
+            // skip transformers not in the main connected component
+            if (config.isExportMainCCOnly() && !EchUtil.isInMainCc(twt, config.isNoSwitch())) {
+                LOGGER.warn(SKIPPING_NOT_IN_MAIN_COMPONENT, "TwoWindingsTransformer", twt.getId());
+                continue;
+            }
+
+            ConnectionBus bus1 = ConnectionBus.fromTerminal(twt.getTerminal1(), config, fakeNodes);
+            ConnectionBus bus2 = ConnectionBus.fromTerminal(twt.getTerminal2(), config, fakeNodes);
+            //do not export the Transformer if bus1==bus2
+            if (EchUtil.isSameConnectionBus(bus1, bus2)) {
+                LOGGER.warn("skipping Transformer: {};  bus1 is equal to bus2: {}", twt, bus1 != null ? bus1.getId() : bus1);
+                continue;
+            }
+
+            EsgBranchConnectionStatus status = getStatus(bus1, bus2);
+
+            //...IIDM gives no rate value. we take rate = 100 MVA But we have to find the corresponding pcu, pfer ...
+            float rate = 100.f;
+
+            //**************************
+            //*** LOSSES COMPUTATION *** (Record 1)
+            //**************************
+
+            double nomiU2 = twt.getTerminal2().getVoltageLevel().getNominalV();
+
+            //...mTrans.getR() = Get the nominal series resistance specified in Ω at the secondary voltage side.
+            double rpu2 = (twt.getR() * parameters.getSnref()) / nomiU2 / nomiU2;  //...total line resistance  [p.u.](Base snref)
+            double gpu2 = ((config.isSpecificCompatibility() ? twt.getG() / 2.0 : twt.getG()) / parameters.getSnref()) * nomiU2 * nomiU2;  //...semi shunt conductance [p.u.](Base snref)
```

Comment:
Do we need to use the `specificCompatibility` parameter provided by the `LoadFlowParameters` ?